task/index.html

```diff
@@ -97,6 +97,43 @@
           />
           <span id="linksDisplay" class="display"></span>
         </div>
+        <div class="inputBox">
+          <label for="">Skills: </label>
+          <div class="multi-select-container" data-testid="skills-multi-select">
+            <button
+              type="button"
+              class="multi-select-button"
+              data-testid="skills-select-button"
+            >
+              <span class="placeholder">Select skills</span>
+              <div class="selected-items"></div>
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                width="24"
+                height="24"
+                viewBox="0 0 24 24"
+                fill="none"
+                stroke="currentColor"
+                stroke-width="2"
+                stroke-linecap="round"
+                stroke-linejoin="round"
+                class="dropdown-icon"
+              >
+                <path d="m6 9 6 6 6-6"></path>
+              </svg>
+            </button>
+            <div class="popover">
+              <input type="text" class="search-input" placeholder="Search..." />
+              <div class="options-list">
+                <div class="option" data-value="select-all">
+                  <span class="option-label">(Select aAll)</span>
```

Comment:
Should it be '(Select All)' instead of '(Select aAll)'?

Reply:
typo, will fix it